README.md

```diff
@@ -17,23 +17,24 @@ npm install -g node-pre-gyp-github
 ```
 
 ## Configuration
-This module is intended to be used with node-pre-gyp. Therefore, be sure to configure and install node-pre-gyp first. After having done that, within **```package.json```** update the ```binary``` property ```host``` so it matches the following format:
+This module is intended to be used with node-pre-gyp. Therefore, be sure to configure and install node-pre-gyp first. After having done that, within **```package.json```** update the ```binary``` properties ```host``` and ```remote_path``` so it matches the following format:
 
 ```
-https://github.com/{owner}/{repo}/releases/download/{1.0.1}
+  "host": "https://github.com/[owner]/[repo]/releases/download/",
+  "remote_path": "{version}"
 ```
-Be sure to replace ```{owner}```, ```{repo}```, ```{1.0.1}``` with actual values.
 
-***WARNING: Variable substitutions are not supported on the ```host``` property so you will have to manually update the version number with every change.*** Failure to do so will result in users installing the wrong binary versions.
+Be sure to replace ```[owner]```, ```[repo]```, with actual values,
+but DO NOT replace ```{version}``` with actual version.
 
-**Tip:** Since the version number will be included within the ```host``` you can ommit it from the package name.
+***WARNING: Variable substitutions are not supported on the ```host``` property but are supported in ```remote_path```, but the value of ```remote_path``` after substitutions must match the release tag name.*** Otherwise it will result in users installing the wrong binary versions.
```

Comment:
I don't think we should call it variable substitutions. This might give someone the impression that they can add one of the other [node-pre-gyp defined variables](https://github.com/mapbox/node-pre-gyp#versioning) here. We should leave the warning to indicate that variable substitutions are not supported on either property host or remote_path with the exception of {version} on remote_path. We should also make it clear that remote_path is not to be used the same way as define in node-pre-gyp. In other words, one can not put a file path consisting of more than one folder because this will not work with Github releases.